server/routes/vendor.router.js

```diff
@@ -0,0 +1,111 @@
+const express = require('express');
+const pool = require('../modules/pool');
+const router = express.Router();
+
+// ROUTES AT /api/vendor/all
+router.get('/all', (req, res) => {
```

Comment:
**CHANGE REQUEST**
Do we require authorization on this endpoint?

Reply:
Completed - John Shands